tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "noEmit": true
```

Comment:
```suggestion
```

Reply:
it's for remove error in tsconfig.json file : 
<img width="877" alt="Capture d’écran 2024-07-22 à 00 17 24" src="https://github.com/user-attachments/assets/be2fbf7f-f0ac-45ad-a7b9-df9be87c7d66">